### ISSUE TITLE
[JENKINS-4492] Control order of trigger execution

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/parameterized-trigger-plugin#437]
<p>I tried to use the parameterized trigger to setup a<br/>
build environment of different application which depends on each other.</p>

<p>I have the following:</p>
<ul class="alternate" type="square">
	<li>build for app 1</li>
	<li>build for app 2</li>
	<li>build for app 3</li>
	<li>publish results</li>
</ul>

<p>The problem is build should be done in order app 1 => app 2 => app 3<br/>
They have dependency <br/>
Currently all of them executed in parallel and it is not possible to<br/>
control the order</p>

<p>Hudson Parameterized Trigger plug-in - Version: 2.0</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=abelous">abelous</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-4492" target="_blank">Control order of trigger execution</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=huybrechts">huybrechts</a>
<li><b>status</b>: Closed
<li><b>priority</b>: Major
<li><b>component(s)</b>: parameterized-trigger-plugin
<li><b>resolution</b>: Won't Fix
<li><b>resolved</b>: 2009-11-27T14:18:57+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 0
<li><b>imported</b>: 2025-12-07
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>I tried to use the parameterized trigger to setup a
build environment of different application which depends on each other.</p>

<p>I have the following:</p>
<ul class="alternate" type="square">
	<li>build for app 1</li>
	<li>build for app 2</li>
	<li>build for app 3</li>
	<li>publish results</li>
</ul>

<p>The problem is build should be done in order app 1 =&gt; app 2 =&gt; app 3
They have dependency 
Currently all of them executed in parallel and it is not possible to
control the order</p>

<p>Hudson Parameterized Trigger plug-in - Version: 2.0</p></pre>
</details>
</details>
<ul><li><i>environment</i>: <code>Platform: All, OS: All</code></li></ul>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>huybrechts:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-4492?focusedId=133652&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-133652">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>You need to configure this differently. Project 1 should trigger project 2, project 2 triggers project 3, ...</p>
</pre>
</details>

</ul>
</details>

<p>You need to configure this differently. Project 1 should trigger project 2, project 2 triggers project 3, ...</p>
